pom.xml

```diff
@@ -100,13 +100,15 @@
     </dependency>
     <dependency>
       <groupId>org.apache.maven</groupId>
-      <artifactId>maven-plugin-api</artifactId>
-      <version>${maven-plugin-api}</version>
+      <artifactId>maven-model</artifactId>
+      <version>3.9.2</version>
+      <scope>provided</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.maven</groupId>
-      <artifactId>maven-project</artifactId>
```

Comment:
I also have one more question. Why is `maven-project` deleted?

Reply:
[`maven-project` was moved to `maven-core`](https://mvnrepository.com/artifact/org.apache.maven/maven-project).